### PR TITLE
Enhancement/15 use aesjs

### LINKED ISSUE #15: Settings | Use external library for AES
Let's switch to this external library https://github.com/ricmoo/aes-js. It has a similar license...

### PR DESCRIPTION
**Pull request by Saidmurod1996**
Fixes #15 

AES encrypted sample data can be found here: https://run.mocky.io/v3/9e62cc89-eb47-46b6-90da-bfc4c212de7e with "password" key.

### CODE REVIEW

File: src/js/shared/config_file.js [hunk 1]
```diff
@@ -30,6 +30,21 @@ const SearchProv = {
 };
 
 const ConfigFile = {
+  decrypt: function (data, key) {
+    // Convert string key to bytes and make it multiple of 16 bytes.
+    key = aesjs.padding.pkcs7.pad(aesjs.utils.utf8.toBytes(key));
```
Comment (jseadragon) [inline src/js/shared/config_file.js:35]: > All keys must be 128 bits (16 bytes), 192 bits (24 bytes) or 256 bits (32 bytes) long.
> from: https://www.npmjs.com/package/aes-js#keys

Shouldn't your `key` be 128, 192, or 256 bits long? I suspect this code is padding with Zeroes which probably not what we want here. 

You should also specify an IV. See the docs here: https://www.npmjs.com/package/aes-js#cbc---cipher-block-chaining-recommended

**Review by jfmdev — APPROVED** [review]

**Comment by pushpendraKh** [comment]
With your given encrypted content, it is failing to update config because of some error in JSON file.
error_options

And with [this URL](https://first-static-website-hosting.s3.ap-south-1.amazonaws.com/enc.txt) where the content was encrypted using Gibberish-aes, decryption fails...

gibberish

**Comment by Saidmurod1996** [comment]
@pushpendraKh now it should work for both:
GibberishAes encrypted data - https://run.mocky.io/v3/ddf1e1cb-d506-40af-bafe-8de001ba69aa
AesJs encrypted data - https://run.mocky.io/v3/9e62cc89-eb47-46b6-90da-bfc4c212de7e
Key - "password"

**Comment by pushpendraKh** [comment]
Works now.

**Review by jfmdev — APPROVED** [review]

**Pull request merged**